tests/unittests/objects/failure_rule/test_firewatch_objects_failure_rule_matches_failure.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (C) 2023 Red Hat, Inc.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+import pytest
+
+from cli.objects.failure import Failure
+from cli.objects.failure_rule import FailureRule
+
+
+@pytest.fixture(autouse=True)
+def setup_tests(assert_default_jira_project_in_env):
```

Comment:
Please add code (`assert` should be here and not in the fixture)

And I think you should remove this, since you set the env anyway